src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -51,7 +51,14 @@ struct rocblas_gemm
     argument
     compute(context& ctx, const shape& output_shape, const std::vector<argument>& args) const
     {
-        gemm(ctx, output_shape, args, op.alpha, op.beta, int8_x4_format);
+        if(!std::strcmp(op.name(), "dot"))
```

Comment:
This can just be `if(op.name() == "dot")`. I think the current form is also generating a tidy warning.

Reply:
Yea it should be`if(op.name() == "dot")`.

Reply:
Change undone.